lib/reek/smells/uncommunicative_parameter_name.rb

```diff
@@ -60,10 +56,9 @@ def examine_context(ctx)
 
       private
 
-      def bad_name?(ctx, name)
-        sanitized_name = sanitize name
-        return false if sanitized_name == '*' || accept_names(ctx).include?(sanitized_name)
-        reject_patterns(ctx).any? { |pattern| sanitized_name.match pattern }
```

Comment:
I agree – I was for the `FIXME` when there was one or two of them, but now when there’s more we’ll remember to do a blanket fix (or stumble upon one of them and recall). :)